src/index.tsx

```diff
@@ -5,6 +5,13 @@ import './index.css';
 import App from './components/App';
 import * as serviceWorker from './serviceWorker';
 
+const org = process.env.REACT_APP_AUTH_PROVIDER;
+switch (org) {
+  case 'stfc':
+    require('./stfc-stylesheet.css');
```

Comment:
Is there any way to move this logic to `theme.ts`?

Reply:
I couldn't quite get this working unfortunately. I had a function in the theme to get the styling, which returned a require import for the CSS. 

The main problem I found was that the import appeared unused, even though it did seem to be calling the function (and I think might have worked). It wouldn't compile though and there's no typescript eslint rule that I could find to override this. I'm also unsure if this would actually have worked when the get styling function returns undefined (in the case of the ESS).